code/game/atoms.dm

```diff
@@ -16,6 +16,8 @@
 
 	var/resistance_flags = NONE
 
+	var/xenotrenchprotection = FALSE
+
```

Comment:
This should absolutely not be on /atom and instead be a bool on caste datums or a caste flag

Reply:
Moved it to /mob/living/carbon/xenomorph